web.test.tests/web.test.tests/Pages/DepositPage.cs

```diff
@@ -71,6 +74,13 @@ public SettingsPage OpenSettingsPage()
             return new SettingsPage(driver);
         }
 
+        public HistoryPage OpenHistoryPage()
+        {
+            History.Click();
+            new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id(HistoryPage.ClearBtnId)));
+            return new HistoryPage(driver);
```

Comment:
Are you sure you need this 'wait'? If yes, I suggest move it to a method 'WaitForLoaded' (or something like this) in the HistoryPage class.

```suggestion
            var historyPage = new HistoryPage(driver);
            historyPage.WaitForLoaded();
            return historyPage;
```

Reply:
No need 'wait' for now.